CODEOWNERS

```diff
@@ -190,8 +190,9 @@ extensions/filters/http/oauth2 @derekargueta @snowp
 # Config Validators
 /*/extensions/config/validators/minimum_clusters @adisuissa @htuch
 # File system based extensions
-/*/extensions/common/async_files @mattklein123 @ravenblack
-/*/extensions/filters/http/file_system_buffer @mattklein123 @ravenblack
+/*/extensions/common/async_files @mattklein123 @ravenblackx
+/*/extensions/filters/http/file_system_buffer @mattklein123 @ravenblackx
+/*/extensions/http/cache/file_system_http_cache @mattklein123 @ravenblackx
```

Comment:
IIRC extensions need to have owners that are envoy maintainers. Probably you should make me the maintainer-owner of the file-system-http-cache directory?

It also looks like a bug below that @tyxia is an owner of gcp_authn even if he created it (at least until he becomes a maintainer).

@mattklein123 do I have the semantics of that right?

Reply:
Yea, I noticed this error (actually throughout this file) when I added code owner. I was thinking it will be fine as long as a maintainer (yanalvlasov) co-owns that file.

Going forwarding, I am wondering if the pattern that author(s) + one maintainer co-own the file is ok for CODEOWNER here?

Reply:
@tyxia that sgtm but I think there's something mechanical in play here flagging that file has having errors if there are non-maintainers listed at all.

Reply:
@jmarantz  I looked into this mechanical thing a bit:
1) The error message (unknown error: make sure xx exists and has write access) you see on the web page here is Github Repo CODEOWNERS syntax . More details can be found [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax) 

2) Envoy has its own policy and check in check_format.py. For example, the extension require at least 2 owners in and requires at least one maintainer OWNER. 
You can find the checks in this [function](https://github.com/envoyproxy/envoy/blob/c9d2507ab6eb9c69a958a0472404c8ee169a4080/tools/code_format/check_format.py#L1110)

So it is really the Github mechanical thing. We could raise the PR to Github community though like [this](https://github.com/orgs/community/discussions/29982)

Reply:
OK I think we can just not worry about these warnings for now, as they don't reflect Envoy policy.

Thanks for investigating, Tianyu!